comp/core/tagger/telemetry/telemetry.go

```diff
@@ -61,6 +61,13 @@ type Store struct {
 	// notification with a group of events.
 	Receives telemetry.Counter
 
+	// ExternalDataRequests tracks the number of requests to the tagger
+	// to generate a container ID from external data.
+	ExternalDataRequests telemetry.Counter
+
+	// ExternalDataSuccess tracks the number of successful resolutions of ExternalData.
+	ExternalDataSuccess telemetry.Counter
```

Comment:
Should this be a single metric with a label that indicates if it was successful or not?

Reply:
Yes I was thinking just a counter that would be incremented in case of success.